lib/puppet/provider/package/rpm.rb

```diff
@@ -310,7 +315,9 @@ def rpmvercmp(str1, str2)
   def insync?(is)
     return false if [:purged, :absent].include?(is)
     should = resource[:ensure]
-    0 == rpm_compareEVR(rpm_parse_evr(should), rpm_parse_evr(is))
+    is.split(self.class::MULTIVERSION_SEPARATOR).any?{ |i|
+      0 == self.rpm_compareEVR(rpm_parse_evr(should), rpm_parse_evr(i))
```

Comment:
Please use a `do..end` block style and a more meaningful variable name instead of `i`